scenario/elasticreload.go

```diff
@@ -158,10 +164,18 @@ func (settings ElasticReloadSettings) execute(sessionState *session.State, actio
 		return
 	}
 
+	timeoutChan := make(<-chan time.Time) // Dummy channel in case no timeout defined
+	if settings.Timeout > 0 {
+		timeoutChan = time.After(time.Duration(settings.Timeout))
+	}
+
 	reloadID := postReloadResponse.ID
 forLoop:
 	for {
 		select {
+		case <-timeoutChan:
+			actionState.AddErrors(errors.New("timeout waiting on reload result event"))
```

Comment:
Thinking maybe we should call `checkStatusOngoing` or otherwise print the last known state when this timeout is reached? That way it's easier to see if something is messed up with the events or if it's just taking a really long time.

Reply:
hmm good point.